content/en/blog/2023/sig-release-spotlight/sig-release-spotlight.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog
+title: "Spotlight on SIG Release (Release Team Subproject)"
+date: 2023-12-01
```

Comment:
This date is in the past. Let's pick a new date; ideally, one that Release Comms are OK with.

/hold

OK to unhold once the publication date is in the future.

Reply:
cc: @fsmunoz ^^